app/controllers/organizations/default_pet_tasks_controller.rb

```diff
@@ -1,7 +1,54 @@
 class Organizations::DefaultPetTasksController < Organizations::BaseController
   layout "dashboard"
 
+  before_action :verify_organization_for_current_user, only: [:new, :create, :edit, :update, :destroy]
+
   def index
     @default_pet_tasks = DefaultPetTask.all
   end
+
+  def new
+    @task = DefaultPetTask.new
+  end
+
+  def create
+    @task = DefaultPetTask.new(task_params)
+
+    if @task.save
+      redirect_to default_pet_tasks_path, notice: "Default pet task saved successfully."
+    else
+      flash.now[:alert] = "Error creating default pet task."
+      render :new, status: :unprocessable_entity
+    end
+  end
+
+  def edit
+    @task = DefaultPetTask.find_by(id: params[:id])
+    raise ActiveRecord::RecordNotFound if @task.nil?
+  end
+
+  def update
+    @task = DefaultPetTask.find(params[:id])
+
+    if @task.update(task_params)
+      redirect_to default_pet_tasks_path, notice: "Default pet task updated successfully."
+    else
+      render :edit, status: :unprocessable_entity
+    end
+  end
+
+  def destroy
+    @task = DefaultPetTask.find(params[:id])
+    @task.destroy
+
+    redirect_to default_pet_tasks_path, notice: "Default pet task was successfully deleted."
+  rescue ActiveRecord::RecordNotFound
+    redirect_to default_pet_tasks_path, alert: "Failed to delete default pet task."
```

Comment:
Typically in these scenarios, applications simply display a 404 page. This usually occurs when a user attempts to delete a record that has already been deleted by someone else moments earlier. Presenting a warning in such cases might be confusing, as users will observe that the pet has in fact been removed from their list.

I think we can simply get rid of this `rescue` and let Rails respond with 404, wdyt? :)

Reply:
[Peek 2024-01-19 13-42.webm](https://github.com/rubyforgood/pet-rescue/assets/33605808/a25fece1-2022-4d88-b8e2-4490cd68e6f2)

Due to Turbo it will rerender the current page regardless, just without the flash message
![image](https://github.com/rubyforgood/pet-rescue/assets/33605808/225733d7-feaf-49ce-a122-ac0c1b923a36)

Reply:
Ok, let's leave it as is then 👍